spec/models/activerecord_obstacle_course_spec.rb

```diff
@@ -349,7 +349,7 @@
     Bullet.start_request
 
     # ------------------------------------------------------
-    orders = Order.all # Edit only this line
+    orders = Order.includes(:items) # Edit only this line
```

Comment:
@s-espinosa @carmer the video posted didn't load for me, can you point me towards any other resources that explain why this worked? I just did what the terminal told me to do when the test failed. Thanks!